test/TestApps/ManualTestApp/main.cpp

```diff
@@ -14,15 +14,35 @@ using namespace winrt::Microsoft::Windows::AppLifecycle;
 
 using namespace std::chrono;
 
-bool IsPackagedProcess()
+HWND g_window = NULL;
+wchar_t g_windowClass[] = L"TestWndClass"; // the main window class name
+
+ATOM _RegisterClass(HINSTANCE hInstance);
+BOOL InitInstance(HINSTANCE, int);
+LRESULT CALLBACK WndProc(HWND, UINT, WPARAM, LPARAM);
+
+std::wstring GetFullIdentityString()
```

Comment:
'GetFullIdentityString' is `GetCurrentPackageFullName but return as a std::wstring`?

Even better if added to `dev\common\AppModel.Identity.h`

```
namespace appModel::Identity
{
inline bool IsPackagedProcess()
{
    ...
}

inline std::wstring GetCurrentPackageFullName()
{
    WCHAR packageFullName[PACKAGE_FULL_NAME_MAX_LENGTH + 1]{};
    UINT32 n{ static_cast<UINT32>(ARRAYSIZE(packageFullName)) };
    const auto rc{ ::GetCurrentPackageFullName(&n, packageFullName) };
    if (rc == APPMODEL_ERROR_NO_PACKAGE)
    {
        return std::wstring();
    }
    THROW_IF_WIN32_ERROR(rc);
    return std::wstring(packageFullName);
}
...
```